content/solana-nft.md

```diff
@@ -0,0 +1,618 @@
+# Creating Solana NFTs With Metaplex
+
+# Lesson Objectives
+
+*By the end of this lesson, you will be able to:*
+
+- Understand the role of Metaplex in the Solana NFT ecosystem
+- Use Candy Machine v2 to create an NFT collection
+- Use a frontend UI to mint NFTs from a Candy Machine
+
+# TL;DR
+
+- **Non-Fungible Tokens (NFTs)** are represented on Solana as SPL Tokens with an associated metadata account, 0 decimals, and a maximum supply of 1
+- **Metaplex** is a collection of tools that simplify the creation and distribution of NFTs on the Solana blockchain
+- **Candy Machine v2** is an NFT distribution program by Metaplex used to mint NFTs from a collection
+
+# Overview
+
+Solana Non-Fungible Tokens (NFTs) are SPL tokens created using the Token Program. These tokens, however, also have an additional metadata account associated with each token mint. This allows for a wide variety of use cases for tokens. You can effectively tokenize anything, from game inventory to art. 
```

Comment:
@jamesrp13 2nd sentence sugg: "Unlike other SPL tokens, however, NFTs also have..."

Reply:
Also, there's a trailing whitespace at the end of this line